shell/browser/native_window_mac.mm

```diff
@@ -489,6 +489,11 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
   [window_ close];
 }
 
+void NativeWindowMac::NotifyWindowClosed() {
```

Comment:
There are some code using NSWindow after `shell_->NotifyWindowClosed()` in `windowWillClose`, I think moving `shell_->NotifyWindowClosed()` to the end of the call should be able to work.